python/dgl/graphbolt/dataloader.py

```diff
@@ -226,6 +226,9 @@ def __init__(
         # separate thread.
         if torch.cuda.is_available():
             copiers = dp_utils.find_dps(datapipe_graph, CopyTo)
+            assert (
```

Comment:
Do we believe this is always true? Is it possible that when user customize graphbolt's feature fetcher, such that they have copyto to copy sampled graph first and then copy feature feature later?

Reply:
That would happen only if they have 2 branches that they are combining, not a single chain of datapipes İ feel. Even then, they can combine the chains and then use a single copyto.

Reply:
Our featurefetcher now copies features to the gpu when ids are on the gpu for all our supported feature classes.

Reply:
Probably just a warning is enough, don't make it a deal breaker.